hibike/hibike_message.py

```diff
@@ -57,9 +57,11 @@
   "GenericError"        : 0xFF
 }
 
+MAX_PACKET_SIZE = 64
```

Comment:
I vaguely remember @nikitakit mentioning 64 bytes was some kind of limit but I may be wrong. If we wanted to support a 64 byte payload we need 1 + 1 + 1 + 1 + 64 + 1 + 1 = 70 bytes. This is because cobs encoding adds a 0 byte, a redundant length field, and 1 byte of overhead.